ui/components/MeshModelRegistry/MeshModelDetails.js

```diff
@@ -27,6 +27,8 @@ import {
 import _ from 'lodash';
 import { JustifyAndAlignCenter } from './MeshModel.style';
 import { withSuppressedErrorBoundary } from '../General/ErrorBoundary';
+import ReactJson from 'react-json-view';
```

Comment:
why we using a new package. Can we use the existing codemirror package to use here?

Reply:
@aabidsofi19, recommend using this library.

Reply:
yeah , we use it in extensions already